### ISSUE TITLE
spurious Person !

### ISSUE THREAD

**Issue by DickBaker** [Wintellect/WintellectWebinars#9]
the repo for Demo4 includes Person.cs which is not actually in the project
- IMHO clutter should be removed [just saying !]

**Comment by DickBaker** [comment]
btw that Demo04 project is in the "2017-11-02 Concurrent Programming in .NET" subfolder

**Comment by DickBaker** [comment]
sorry, closed in error (finger/brain/coord issue!)

**Comment by JohnWintellect** [comment]
Hey @bellj, can I remove this file without breaking anything in the demos? Thanks!

**Comment by bellj** [comment]
Yes, the Person.cs file can be safely removed from the Demo04 folder. It was added to answer a question asked during a previous demonstration. It was later removed from the project but never actually deleted from disk.
